sherpa/astro/xspec/src/_xspec.cc

```diff
@@ -1293,10 +1308,11 @@ static PyMethodDef XSpecMethods[] = {
   XSPECMODELFCT_CON(C_PartialCovering, 1),
   XSPECMODELFCT_CON(C_rdblur, 4),
   XSPECMODELFCT_CON(C_reflct, 5),
+  XSPECMODELFCT_CON_F77(rgsxsrc, 1),
   XSPECMODELFCT_CON(C_simpl, 3),
   XSPECMODELFCT_CON(C_zashift, 1),
   XSPECMODELFCT_CON(C_zmshift, 1),
-  
+
```

Comment:
Hmmm, why did I not add commented-out lines saying

    XSPECMODELFCT_CON(C_clumin, 4),

for the commented-out convolution models? (note that in the comments `C_lumin` is wrong and should be `C_clumin`).